.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Generally we don't want to be adding files from `.pytest_cache/`.  To avoid this we could add `.pytest_cache/*` to the `.gitinore` or just be careful when using `git add`.